### PR TITLE
fixing header change on lang change

### PR DESCRIPTION
**Pull request by sendilkumarn**
This fix adds change of header text when the lang is changed.

### CODE REVIEW

File: generators/client/templates/angular/src/main/webapp/app/shared/language/_language.helper.ts [hunk 1]
```diff
@@ -23,25 +24,30 @@ export class JhiLanguageHelper {
...
+            this.updateTitle();
+        });
+
+        this.translateService.onLangChange.subscribe((event: LangChangeEvent) => {
+            this.updateTitle();
+        });
+    }
+
+    private getPageTitle(routeSnapshot: ActivatedRouteSnapshot) {
+        let title: string = (routeSnapshot.data && routeSnapshot.data['pageTitle']) ? routeSnapshot.data['pageTitle'] : 'ang4App';
```
Comment (jdubois) [inline generators/client/templates/angular/src/main/webapp/app/shared/language/_language.helper.ts]: I'll modify this: if there no title, let's not put anything (better than a wrong name)

Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/app/shared/language/_language.helper.ts]: my bad. Sorry 👍

Comment (deepu105) [inline generators/client/templates/angular/src/main/webapp/app/shared/language/_language.helper.ts]: We could set the app name as default title right

Comment (jdubois) [inline generators/client/templates/angular/src/main/webapp/app/shared/language/_language.helper.ts]: I put an empty String, at least nous one will complain it's wrong!

Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/app/shared/language/_language.helper.ts]: I fixed it via [this](https://github.com/jhipster/generator-jhipster/commit/11495631ff45c1f51f5083f2b6e8b9f970972467)

**Pull request merged**